assets/public/js/main.js

```diff
@@ -6,3 +8,93 @@ function toggleCartDropdown() {
 }
 
 cartBtn.onclick = toggleCartDropdown;
+
+/* Navigation menus */
+
+const mainNavButtonId = "navBtn";
+const featuresSubNavButtonId = "featuresBtn";
+const backButtonId = "backBtn";
+
+// Constructor for NavMenu object.
+function NavMenu(element, showClasses, hideClasses) {
+  this.element = element;
+  this.showClasses = showClasses.split(" ");
```

Comment:
Isn't it possible to toggle the classes without splitting?

Reply:
Simplified.